src/test/json2xlsx.js

```diff
@@ -82,51 +82,83 @@ describe('json2xlsx', () => {
       const sheets = XLSX.readFile('./temp/ccd-definitions.xlsx').Sheets;
       assert(Object.keys(sheets).length > 0, 'No sheets have been created');
 
+      const assertCell = (sheetName, cell, expectedCellValue) => {
+        const actualCellValue = expectedCellValue === undefined ? sheets[sheetName][cell] : sheets[sheetName][cell].v;
+        assert.equal(actualCellValue, expectedCellValue, `Unexpected value found in ${cell} cell of ${sheetName} sheet`);
```

Comment:
assert.equal is deprecated, the docs advise the use of `const assert = require('assert').strict;` instead of `const assert = require('assert');` or at least `assert.strictEqual()`